lab-segmentation.cpp

```diff
@@ -88,130 +100,148 @@ void lab11()
     }
 
     // Extract pixel features from the image.
-    cv::Mat feature_image = extractFeatures(frame);
-
-    if (model)
```

Comment:
Dette er nok her for at studentene skal kunne kompilere og kjøre (vise video) uten at oppgavene er ferdig løst. Derfor er det mulig at vi vil ha noe slikt tilbake igjen. Vi får se hvordan det tar seg ut når vi lager oppgaven

Reply:
Det kjører hos meg nå, gjorde det som i python